lib/single-new-eks-opensource-observability-construct/graviton-index.ts

```diff
@@ -0,0 +1,89 @@
+import { Construct } from 'constructs';
+import { EksBlueprint, utils } from '@aws-quickstart/eks-blueprints';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { GrafanaOperatorSecretAddon } from './grafanaoperatorsecretaddon';
+import * as amp from 'aws-cdk-lib/aws-aps';
+import * as eks from 'aws-cdk-lib/aws-eks';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as assert from "assert";
+
+export default class SingleNewEksOpenSourceGravitonObservabilityConstruct {
+    constructor(scope: Construct, id: string) {
+        // AddOns for the cluster
+        const stackId = `${id}-observability-accelerator`;
+
+        const account = process.env.COA_ACCOUNT_ID! || process.env.CDK_DEFAULT_ACCOUNT!;
+        const region = process.env.COA_AWS_REGION! || process.env.CDK_DEFAULT_REGION!;
+        const ampWorkspaceName = process.env.COA_AMP_WORKSPACE_NAME! || 'observability-amp-Workspace';
+        const ampPrometheusEndpoint = (blueprints.getNamedResource(ampWorkspaceName) as unknown as amp.CfnWorkspace).attrPrometheusEndpoint;
+        
+        const amgEndpointUrl = process.env.COA_AMG_ENDPOINT_URL;
+
+        // All Grafana Dashboard URLs from `cdk.json` if present
+        const clusterDashUrl: string = utils.valueFromContext(scope, "cluster.dashboard.url", undefined);
+        const kubeletDashUrl: string = utils.valueFromContext(scope, "kubelet.dashboard.url", undefined);
+        const namespaceWorkloadsDashUrl: string = utils.valueFromContext(scope, "namespaceworkloads.dashboard.url", undefined);
+        const nodeExporterDashUrl: string = utils.valueFromContext(scope, "nodeexporter.dashboard.url", undefined);
+        const nodesDashUrl: string = utils.valueFromContext(scope, "nodes.dashboard.url", undefined);
+        const workloadsDashUrl: string = utils.valueFromContext(scope, "workloads.dashboard.url", undefined);
+
+        Reflect.defineMetadata("ordered", true, blueprints.addons.GrafanaOperatorAddon);
+        const addOns: Array<blueprints.ClusterAddOn> = [
+            new blueprints.addons.AwsLoadBalancerControllerAddOn(),
```

Comment:
code is not aligned to latest standards. Please fix the code per our standards which we introudced friday. You need to use Observability.blueprint.